.circleci/config.yml

```diff
@@ -19,6 +19,21 @@ docker_config_defaults: &docker_config_defaults
     # This IAM user only allows read-write access to ECR
     aws_access_key_id: ${CIRCLECI_AWS_ACCESS_KEY_FOR_ECR_READ_WRITE_V4}
     aws_secret_access_key: ${CIRCLECI_AWS_SECRET_KEY_FOR_ECR_READ_WRITE_V4}
+
+executors:
+  windows-gpu:
+    machine:
+      resource_class: windows.gpu.small.prototype
+      image: windows-server-2019-nvidia:edge
+      shell: bash.exe
+
+  windows-cuda:
```

Comment:
windows-gpu and windows-cuda are somewhat ambiguous names I think. Why not `windows-gpu` and `windows-cpu`/`windows-cpu-with-cuda` or whatever?

Reply:
Sounds good